rsconf/package_data/vm_devbox/main.sh.jinja

```diff
@@ -5,6 +5,7 @@ vm_devbox_main() {
         return
     fi
     yum-config-manager --set-enabled crb
+    ln -s '{{ this.libvirt_d }}' '{{ this.var_libvirt_d }}'
```

Comment:
Do we want this as an install_symlink?

Reply:
Nice. Didn't know about that one... 9fa7476